src/assets/images/icons/icon_menu-kebab.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="3" height="20" fill="none" viewBox="0 0 3 18">
```

Comment:
Is it OK to leave the `width` and `height` here? Other SVG icons don't seem to have them.

Reply:
It seems `h-4.5` in `tailwind.config.ts` is missing.